README.rst

```diff
@@ -163,6 +163,51 @@ Delete
     print(response)
     # <class pybutton.Response >
 
+Accounts
```

Comment:
A section in the readme talking about the nature of our `pybutton.Response` class is worthwhile at this point, I think!